### ISSUE TITLE
Dependency System.Net.Http' 4.3.0 has a known high severity vulnerability

### ISSUE THREAD

**Issue by OnurGumus** [akkadotnet/HOCON#393]
When I install hocon it depends on netstandard.library which depends on System.Net.Http' 4.3.0. Perhaps it is time to move to some newer .net version for this library unless we live with potentially unsecure lib and pestering warning

`
Warning NU1903: Package 'System.Net.Http' 4.3.0 has a known high severity vulnerability
`

**Comment by Aaronontheweb** [comment]
Yep, we just fixed a bunch of these in Akka.Hosting - I suppose all of these libraries need updating too
